packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.kt

```diff
@@ -419,6 +671,7 @@ class FileDownloader(
       .url(assetEntity.url!!.toString())
       .addHeadersFromJSONObject(assetEntity.extraRequestHeaders)
       .addHeadersFromJSONObject(extraHeaders)
+      .header("Accept", "$DIFF_CONTENT_TYPE,*/*")
```

Comment:
see note above about maybe only adding this for launch asset request since we asset that only the launch asset response can be a diff (defense against poorly-implemented servers)